SonarQube.MSBuild.Tasks/AttachBuildWrapper.cs

```diff
@@ -28,7 +28,6 @@ public class AttachBuildWrapper : Task
         private const string BuildWrapperExeName64 = "build-wrapper-win-x86-64.exe";
         private const string AttachedBinaryFileName32 = "interceptor32.dll";
         private const string AttachedBinaryFileName64 = "interceptor64.dll";
-        private const string BuildWrapperSubDirName = "build-wrapper-win-x86";
```

Comment:
I changed the task: it now expects to be told exactly where to find the build wrapper binaries, rather than looking for them in a particular subdirectory under the supplied location. This is to enable the user to set a property so specify precisely where the targets should be found.